atomic_reactor/plugins/exit_koji_promote.py

```diff
@@ -508,7 +509,7 @@ def add_log_type(output):
                     'parent_id': parent_id,
                     'repositories': repositories,
                     'tags': list(tags),
-                    'config': config
+                    'config': config,
```

Comment:
Not strictly needed for this change (although I prefer to have a trailing comma after each entry anyway).